.changeset/lazy-lemons-move.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-docs/gatsby-theme-docs': minor
+'@commercetools-website/documentation': minor
+'@commercetools-docs/ui-kit': minor
+---
+
+Created a wrapper component for using lord-icon icons
```

Comment:
I see that there are two changesets here. Can you please remove one?

Reply:
done. Thank you @timonrey